.drone.yml

```diff
@@ -0,0 +1,25 @@
+---
+kind: pipeline
+type: docker
+name: default
+
+steps:
+  - name: build
+    image: abakus/statuttbuilder:latest
+    commands:
+      - ./generate.sh
+    when:
+      event:
+        - pull_request
+
+  - name: push
```

Comment:
What is this doing?

Reply:
It doesn't work yet, but it's supposed to do the same things as `Travis` does, where it builds then pushes the public code to another branch, which is served.

The whole process isn't planned yet.